packages-exp/functions-exp/src/api.ts

```diff
@@ -83,10 +83,31 @@ export function httpsCallable<RequestData = unknown, ResponseData = unknown>(
   functionsInstance: Functions,
   name: string,
   options?: HttpsCallableOptions
+): HttpsCallable<RequestData, ResponseData> {
+  const instance = getModularInstance<FunctionsService>(
+    functionsInstance as FunctionsService
+  );
+  const url = instance._url(name);
+
+  return _httpsCallable<RequestData, ResponseData>(instance, url, options);
+}
+
+/**
+ * Returns a reference to the callable https trigger with the given URL.
```

Comment:
In the guide docs we call these "HTTP triggers."  Unless it's a literal, it may as well be uppercase I figure, "S" or no.

Looks like there are a few other instances to look at in this PR.